### DISCUSSION TITLE
Moving our discussions to a forum

### DISCUSSION THREAD

**Discussion by yisiliu** [DimensionDev/Maskbook#2620] [General]
I am thinking about moving our discussions, especially non-technical ones, to a forum and the best option in my mind is [Discource](https://github.com/discourse/discourse). Do you guys have any other options? We can enable a Mask login option in the forum to fully utilize the use of a cryptographic key pair as the account.

**Comment by yisiliu** [comment]
Note: This comes with the build of MaskDAO and hopefully this'll help us organize our daily product discussions in a better way.

**Comment by Tsukimoe** [comment]
It seems that Flarum is also great, We need MaskDAO
when do we plan to start this part?

**Comment by yisiliu** [comment]
Finalist:
[Discourse](https://github.com/discourse/discourse)
[NodeBB](https://github.com/NodeBB/NodeBB)
[Flarum](https://github.com/flarum/flarum)
